src/pages/GourmetList/FoodDataList.js

```diff
@@ -0,0 +1,280 @@
+              <Food>
+                <FoodHeader>
+                  <FoodImage src={lists.thumbnail_img} />
+                  <Container>
+                    <FoodInfoHeader>
+                      <InfoHeader>
+                        <InfoHeaderId>{lists.id}.</InfoHeaderId>
+                        <InfoHeaderName>{lists.name}</InfoHeaderName>
+                      </InfoHeader>
+                      <InfoDistance>{lists.distance}</InfoDistance>
+                    </FoodInfoHeader>
+                    <FooterInfoContainerHash>
+                      <Category>{lists.category}</Category>
+                      <Tag>{lists.tag}</Tag>
+                    </FooterInfoContainerHash>
+                    <FooterInfoContainerScore>
+                      <Score>
+                        <ScoreNumber>{lists.score}</ScoreNumber>
+                        <ScoreText>점</ScoreText>
+                      </Score>
+                      <Star>
+                        <img src="images/list_star.png" alt="" />
+                        {lists.star}
+                      </Star>
+                      <Heart>
+                        <img src="images/list_heart.png" alt="" />
+                        {lists.heart}
+                      </Heart>
+                    </FooterInfoContainerScore>
+                  </Container>
+                </FoodHeader>
+                <Review>"{lists.review}..."</Review>
+                <Id>
+                  <Recommend>
+                    <RecommendNumber>36</RecommendNumber>
+                    <RecommendText>명이 추천했습니다.</RecommendText>
+                  </Recommend>
+                  <IdName>by.{lists.id_name}</IdName>
+                </Id>
+              </Food>
+            ))}
+            <BlueButton>
+              <img src="images/list_list.png" alt="" /> 검색 결과 더보기
+            </BlueButton>
+          </FoodInner>
+        </FoodContainer>
+      </FoodOl>
+    </FoodData>
+  );
+};
+
+export default FoodList;
+
+const FoodData = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-color: white;
+  border-radius: 14px;
+`;
+
+const FoodOl = styled.ol`
+  margin: 0px;
+  padding: 0px;
+  list-style: none;
+`;
+
+const FoodContainer = styled.li`
+  /* padding: 0px 20px; */
+  list-style: none;
+`;
+
+const Food = styled.div`
+  padding: 25px 20px;
+  border-bottom: 1px solid rgb(245, 245, 245);
+  :hover {
+    background-color: rgb(245, 245, 245);
+    cursor: pointer;
+  }
+`;
+
+const FoodInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding-top: 20px;
+`;
+
+const FoodHeader = styled.div`
+  display: flex;
+`;
+
+const FoodImage = styled.img`
+  width: 110px;
+  height: 110px;
+  object-fit: cover;
+  border-radius: 8px;
+`;
+
+const Container = styled.div`
+  width: 100%;
+  margin-left: 17px;
+`;
+
+const InfoHeader = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
```suggestion
  display: flex;
  align-items: center;
  justify-content: space-between;
```
mixin 코드는 main브랜치에 rebase and merge 되어있습니다. 내 브랜치로 rebase하면 이제 쓸 수 있습니다. mixin으로 수정할 수 있는 코드는[styled-component](https://www.notion.so/wecode/React-Styled-Components-761ad57c7a6d4ce8a10dc683e1866a8a)문서에 globalStyles 부분 참고하셔서 수정 해 보세요!
* 힌트1. `theme`은 어디서 import 되고있는지 찾아보세요.
* 힌트2. `${({theme})=>{console.log(theme)}}`을 활용 해 보세요

Reply:
수정했습니다!